src/main/java/com/gluonhq/Main.java

```diff
@@ -113,8 +113,10 @@ public Double fromString(String s) {
                 new Separator(Orientation.VERTICAL),
                 fontFamilies,
                 fontSize,
-                actionButton(LineAwesomeSolid.BOLD, editor.getActionFactory().decorate(TextDecoration.builder().fontWeight(FontWeight.BOLD).build())),
-                actionButton(LineAwesomeSolid.ITALIC, editor.getActionFactory().decorate(TextDecoration.builder().fontPosture(FontPosture.ITALIC).build())),
+                createToggleButton(LineAwesomeSolid.BOLD, property -> new DecorateFontWeightAction(editor, property)),
```

Comment:
Run app, type two words, go back, select the first word and apply bold (bold toggle is selected). Then if you move the cursor with the arrows to the next word, bold toggle is deselected as expected.

However:
Run app, type two words, go back, select the first word and apply bold (bold toggle is selected). Then move the cursor _with the mouse_ to the middle of the second word: bold toggle is still selected.

Reply:
This should be fixed with latest commit

Reply:
I still get the same issue. For instance, apply strikethrough to a word, move caret with mouse, strikethrough toggle is still selected. Once you move caret with arrow key it gets fixed.

Reply:
fixed now